src/sentry/models/auditlogentry.py

```diff
@@ -63,18 +78,9 @@ def save(self, *args, **kwargs):
     def get_actor_name(self):
         if self.actor:
             # fix display name if needed
-            uuid_regex = re.compile(
-                r"(.*)([0-9a-fA-F]{6})\-[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{7}"
-            )
-            uuid_match = re.match(uuid_regex, self.actor.get_display_name())
-
-            if uuid_match:
-                uuid_prefix = uuid_match.groups()[1]
-                integration_name = (uuid_match.groups()[0]).replace("-", " ")
-                integration_name = integration_name.title()
-                integration_name = integration_name.replace("Scim", "SCIM")
-
-                return integration_name + " (" + uuid_prefix + ")"
+            scim_match = is_scim_token_actor(self.actor)
+            if scim_match:
```

Comment:
```suggestion
            if is_scim_token_actor(self.actor)
```
since you're only using that variable in one place, and it doesn't help with readability, its usually better to inline the call.

Reply:
ah, i see that you were returning something from `is_scim_token_actor`, in which case this made more sense. i would add though, if you prefix a function with `is`, i'd usually expect that function to return a boolean.